### PR TITLE
Update governance structure overview to include specs as Projects

### PR DESCRIPTION
**Pull request by lehors**
This updates the overview to get aligned with the new structure where spec development happens in a Project.

### CODE REVIEW

File: organizational-structure-overview.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ The OpenSSF is comprised of instances of the following categories of official gr
...
 - A **Technical Deliverable** is any technical content produced by a Technical Initiative, such as open source licensed software, a specification, or a technical guide.
-   - Each **Project** shall have at least one Technical Deliverable including open source licensed software; this is what defines a Project as distinct from other Technical Initiatives such as SIGs.
+   - Each **Project** shall have at least one Technical Deliverable including open source licensed software or specification; this is what defines a Project as distinct from other Technical Initiatives such as SIGs.
    - A **Service** is a publicly-run instance of software as a service, and is another form of Technical Deliverable distinct from the release of open source licensed software. This may be an operational output of a Technical Initiative in which software is either built or acquired to support or automate OSSF transactions.
 
 The following table describes the main types of groups and their characteristics.
```
Comment (steiza) [inline organizational-structure-overview.md:28]: Minor: should we add to the `Project` row of this table the primary output as "software or specification"?

**Review by steiza — APPROVED** [review]
One minor comment; otherwise looks good to me!

**Review by SecurityCRob — APPROVED** [review]

File: organizational-structure-overview.md (hunk 1 above)
Comment (lehors) [reply organizational-structure-overview.md:28]: Good catch! Done.

**Review by di — APPROVED** [review]

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Pull request merged**